test/functional/insertItem.test.ts

```diff
@@ -360,6 +360,47 @@ suite("InsertItem", async (): Promise<void> => {
         }
     }
 }`;
+
+        const oneParameterTemplateInt = `{
+    "parameters": {
+        "parameter1": {
+            "type": "int",
+            "defaultValue": 42
+        }
+    }
+}`;
+        const oneParameterTemplateArray = `{
+    "parameters": {
+        "parameter1": {
+            "type": "array",
+            "defaultValue": []
+        }
+    }
+}`;
+        const oneParameterTemplateBool = `{
+    "parameters": {
+        "parameter1": {
+            "type": "bool",
+            "defaultValue": true
+        }
+    }
+}`;
+        const oneParameterTemplateObject = `{
+    "parameters": {
+        "parameter1": {
+            "type": "object",
+            "defaultValue": {}
+        }
+    }
+}`;
+        const oneParameterTemplateSecureObject = `{
+    "parameters": {
+        "parameter1": {
+            "type": "secureobject",
+            "defaultValue": {}
+        }
+    }
+}`;
         suite("Insert one parameter", async () => {
             await doTestInsertItem(emptyTemplate, oneParameterTemplate, TemplateSectionType.Parameters, ["parameter1", "String", "default", "description"]);
```

Comment:
nit: doTestInsertItem doesn't need to be async, just the functions passed to the test() calls within it.  Personally I prefer the naming createInsertItemTests() to minimize the confusion of a function that calls test() vs one that's expected to be called from a test() body.

Reply:
Fixed.

Reply:
Fixed.